object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,7 @@
+/* exported setValue */
+// create  a variable and set its value to the object at key and then assign that value to the value parameter
+// return the object variable just created
+function setValue(object, key, value) {
+  var Object = (object[key] = value);
```

Comment:
this should be a lowerCase `o`, you don't want to use capital letters for variables except in very special circumstances. Also, `Object` is a class in Javascript which you are currently overwriting by using `Object` as a variable. watch out for those uppercase letters!